recipes/_compression_libs.rb

```diff
@@ -24,7 +24,11 @@
 # Everybody gets snappy
 case node['platform_family']
 when 'debian'
-  pkgs += ['libsnappy1', 'libsnappy-dev']
+  pkgs += if node['platform_version'] >= 16
```

Comment:
you also still need a ``.to_i`` to convert the node attribute to an integer:  ``node['platform_version'].to_i``